### PR TITLE
[WNMGDS-264] - Add pattern for third party external links 

### PR DESCRIPTION
**Pull request by line47**
### Added
Added a react example and a docs page for third party external links. 

#### Screenshots 
Screen Shot 2019-10-30 at 3 11 59 PM
Screen Shot 2019-10-30 at 3 12 04 PM
Screen Shot 2019-10-30 at 3 12 10 PM

### CODE REVIEW

File: packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+*/
+
+/*
+---
+
+Use this pattern when the user is in the middle of a task or application to let them know that the action they are taking will move them away from the current site and or process. This serves as a warning where the user can then decide to remain where they are in the application or process.
+
+### Learn more
+
+- [Making an accessible dialog box](https://www.nczonline.net/blog/2013/02/12/making-an-accessible-dialog-box/)
```
Comment (bernardwang) [inline packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss]: are these the same resources on the dialog page? maybe we could link those cms privacy guidance pages?

Comment (line47) [reply packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss]: Yep, they are the same. I'm not sure which links you are referring to?

Comment (bernardwang) [reply packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss]: Maybe we could replace the resources on dialogs with the links Mel posted on this page:
https://github.com/CMSgov/design-system/issues/499#issuecomment-537659356

Comment (line47) [reply packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss]: Ready for another pass

**Comment by mliupaeter** [comment]
This looks great! Thanks guys!

**Pull request merged**